puts modified and added formulae

ladislas opened the pull request:
kind of like #258, this PR outputs the added and modified formulae to help debug.

I could also add `deleted_formulae` if needed.

Let me know what you think.

sjackman commented:
I scanned the code quickly, and I didn't see that the value of `@formulae @added_formulae @modified_formulae` are printed anywhere. If that's the case, :+1: from me.

iMichka commented:
I like too. Nice addition.

Can we add the deleted formulae too?

MikeMcQuaid commented on cmd/brew-test-bot.rb:
```diff
@@ -574,6 +574,13 @@ def download
+      puts <<~EOS
+
+        Bottles to be built for:
+          added formulae    #{@added_formulae.empty? ? "(empty)" : @added_formulae}
```
Use `blank?` rather than `empty?`

ladislas replied:
sure!

MikeMcQuaid commented on cmd/brew-test-bot.rb:
```diff
@@ -574,6 +574,13 @@ def download
+
+      puts <<~EOS
+
+        Bottles to be built for:
```
Bottles aren't necessarily going to be built. How about `Formula changes to be tested:`

ladislas replied:
Sounds good to me, I'll change that.

ladislas commented:
I just updated the PR based on your feedback.

I've added `deleted_formulae`.

it's gonna print as `["formula1", "formula2"]` -- should it be printed as `formula1, formula2` instead?

sjackman commented:
> it's gonna print as ["formula1", "formula2"] -- should it be printed as formula1, formula2 instead?

Yes, separated with either comma space `, ` or simply space ` `

ladislas commented:
@sjackman should I use:

```ruby
@modified_formulae.join(' ') # => "formula1 formula2"

#or 

@modified_formulae.reject(&:blank?).join(' ') # => same but will remove '' and nil, just in case
```

ladislas commented:
I went with the safer `reject`, to handle all cases.

sjackman commented:
There will not be blank formulae names, so `@modified_formulae.join(' ')` is fine.

MikeMcQuaid commented:
> There will not be blank formulae names, so `@modified_formulae.join(' ')` is fine.

Agreed. Once this is done this should be good to go!

ladislas commented:
Done! ✅ (from my phone for the first time)

sjackman commented:
> Done! ✅ (from my phone for the first time)

Hah. Well done! Thanks again for this work, Ladislas! 🌟

MikeMcQuaid commented:
Thanks again @ladislas!

MikeMcQuaid commented:
@ladislas Please check your email 😂
